lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8829,9 +8829,9 @@ bool SwiftASTContext::GetCompileUnitImportsImpl(
   if (cu_imports.size() == 0)
     return true;
 
-  Progress progress("Getting Swift compile unit imports",
-                    compile_unit->GetPrimaryFile().GetFilename().GetCString(),
-                    cu_imports.size());
+  std::string category = "Importing Swift module dependencies for ";
+  category += compile_unit->GetPrimaryFile().GetFilename();
```

Comment:
Would `main.swift` always be the primary filename here? The category would be "Importing Swift module dependencies for main.swift" which I think works as a category name (as opposed to "Importing Swift module dependencies for"), I just wanna make sure that the filename here wouldn't change.

Reply:
So, in short: yes!